frontend/src/utils/color.ts

```diff
@@ -0,0 +1,16 @@
+// Copied from https://stackoverflow.com/a/59387478
+export const lighten = (color: string, lighten: number) => {
+  const c = color.replace('#', '');
+  const rgb = [
+    parseInt(c.substr(0, 2), 16),
+    parseInt(c.substr(2, 2), 16),
+    parseInt(c.substr(4, 2), 16),
```

Comment:
I discovered that `substr` should be avoided, replaced by `substring`.

```suggestion
    parseInt(c.substring(0, 2), 16),
    parseInt(c.substring(2, 4), 16),
    parseInt(c.substring(4, 6), 16),
```

See: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/String/substr
See: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/String/substring